### PR TITLE
Changed the caption message

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @ACaspeer
- [ ] @AndreiTeradyne
- [ ] @Daduc38
- [ ] @ElMT2023
- [ ] @JGT90
- [ ] @JameHann
- [ ] @JoelUsername
- [ ] @Raccoon137
- [ ] @frehiwotberhe
- [ ] @jgtres
- [ ] @joel8800
- [ ] @kbushe
- [ ] @mccroryn
- [ ] @rbelleau88
- [x] @shtaudor

### PR DESCRIPTION
**Pull request by rbelleau88**
Here is my pull request comment

# My Header

This resolves #3

- [x] #25 Add a caption to my slide
- [] And another task

This update resolves #25

### CODE REVIEW

File: _slides/17-rbelleau88.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (rbelleau88) [inline _slides/17-rbelleau88.md:6]: This is a line that I removed

**Review by Daduc38 — APPROVED** [review]
Code has been properly made as usual :-)

File: _slides/17-rbelleau88.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
+Raoul's awesome caption
```
Comment (Daduc38) [inline _slides/17-rbelleau88.md:6]: Nice Job

**Review by Daduc38 — APPROVED** [review]
Final approval

File: _slides/17-rbelleau88.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (shtaudor) [inline _slides/17-rbelleau88.md:3]: Cool change!

**Review by shtaudor — APPROVED** [review]
Great job!

**Pull request merged**